include/deal.II/matrix_free/cuda_matrix_free.templates.h

```diff
@@ -871,6 +871,9 @@ namespace CUDAWrappers
       return internal::get_conflict_indices<dim, Number>(filter, constraints);
     };
 
+    if (begin == end)
+      return;
+
```

Comment:
Is this meant to be in this PR?

Reply:
Yes, `GraphColoring::make_graph_coloring` complains otherwise.